introduction.md

```diff
@@ -3,3 +3,8 @@ Watashi no ichiban sukina codeschool path ha web-security de gozaimasu.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?